Text based game/Text based game/main.swift

```diff
@@ -0,0 +1,99 @@
+//
+//  main.swift
+//  Text based game
+//
+//  Created by Lilia Yudina on 10/1/19.
+//  Copyright © 2019 Lilia Yudina. All rights reserved.
+//
+
+import Foundation
+
+print("How many hours do you sleep on average?")
+var hoursOfSleep = Int(readLine() ?? "-1") ?? -1
+
+switch hoursOfSleep {
+case 0..<4:
+    print("not enough sleep")
+case 4...6:
+    print("enough hours to not go crazy")
+case 7...8:
+    print("ideal")
+default:
+    print("sleepy head")
+}
+
+if 0...4 ~= hoursOfSleep {
+    print("when do you go to sleep? Choose between: early, late")
+    let response = readLine()
```

Comment:
adding a .lowerCased() function would have allowed for "Early"/ "EARLY" to have worked whereas now the game ends if the user doesn't type "early".